cmd/sidecar/main.go

```diff
@@ -40,17 +46,33 @@ func main() {
 		os.Exit(1)
 	}
 
-	configSecrets, err := secretsConfig.NewFromEnv(tokenFilePath)
+	configStorage, err := storageConfig.NewFromEnv()
 	if err != nil {
 		errLogger.Printf(err.Error())
 		os.Exit(1)
 	}
+	// Create new Storage
+	stor, err := storage.NewStorageHandler(*configStorage)
+
+	//if conjurSecretsDest == 1  &&  authn.Config.ContainerMode != "init" {
+	//      // notify on invalid configuration and exit
+	//      errLogger.Printf(" appropriate message for not supporting sidecar ....")
+	//      os.Exit(1)
+	//}
+
+	if conjurSecretsDest == 1 {
```

Comment:
what is `1`? can we have some const for this?

Reply:
added const to top of file. Also added above the associated code both that this is a TODO